x-pack/plugins/spaces/public/nav_control/components/spaces_menu.tsx

```diff
@@ -121,7 +121,6 @@ class SpacesMenuUI extends Component<Props, State> {
         key={`spcMenuList`}
         data-search-term={searchTerm}
         className="spcMenu__spacesList"
-        hasFocus={this.state.allowSpacesListFocus}
```

Comment:
Unfortunately, this change broke keyboard navigation for the spaces menu 😔 
I don't see a simple change that would fix it, so I opened a separate issue to address that to unblock this PR: #130898